server/src/internalClusterTest/java/org/opensearch/snapshots/SearchableSnapshotIT.java

```diff
@@ -670,6 +672,30 @@ public void testCacheIndexFilesClearedOnDelete() throws Exception {
         logger.info("--> validated that the cache file path doesn't exist");
     }
 
+    public void testIndexCreationWithIndexStoreTypeRemoteStoreThrowsException() {
```

Comment:
Can we make this a unit test in `MetadataCreateIndexServiceTests` instead of an integration test?

Reply:
Sure will add it.